pkg/backend/httpstate/state.go

```diff
@@ -261,8 +261,8 @@ func (b *cloudBackend) newUpdate(ctx context.Context, stackRef backend.StackRefe
 	}, nil
 }
 
-func (b *cloudBackend) getSnapshot(ctx context.Context, stackRef backend.StackReference) (*deploy.Snapshot, error) {
-	untypedDeployment, err := b.exportDeployment(ctx, stackRef, nil /* get latest */)
+func (b *cloudBackend) getSnapshot(ctx context.Context, stk backend.Stack) (*deploy.Snapshot, error) {
+	untypedDeployment, err := b.exportDeployment(ctx, stk, nil /* get latest */, false)
```

Comment:
Was there a reason for this move from `StackReference` to `Stack`?  I thought we moved the other direction on this API recently? 

Also - why `stk`?

Reply:
The file codepath uses Stack not StackReference so I was solidifying on that path - we can of course go to StackReferences. I used stk here as it's used within the filestate codepath